moveit_planners/chomp/chomp_interface/include/chomp_interface/chomp_planning_context.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * chomp_planning_context.h
+ *
+ *  Created on: 27-Jul-2016
+ *      Author: ace
+ */
+
+#ifndef CHOMP_PLANNING_CONTEXT_H_
+#define CHOMP_PLANNING_CONTEXT_H_
+
+#include <moveit/planning_interface/planning_interface.h>
+#include <chomp_interface/chomp_interface.h>
+#include <chomp_interface/chomp_planning_context.h>
+#include <moveit/collision_distance_field/collision_detector_allocator_hybrid.h>
+
+#include <tf/transform_listener.h>
+#include <moveit/planning_scene_monitor/planning_scene_monitor.h>
+
+#include <moveit/robot_state/conversions.h>
+
+namespace chomp_interface {
+
+class ChompPlanningContext: public planning_interface::PlanningContext {
+
+  typedef boost::shared_ptr <ChompPlanningContext> Ptr;
```

Comment:
I used the convention from ROS messages. Usually for every message type you have two Ptr typedefs.